### DISCUSSION TITLE
Provide suggestions of libraries that might already do what you are doing

### DISCUSSION THREAD

**Discussion by mo22** [community/community#7666] [Copilot Conversations]
A couple of times now I had the impression that copilot is pretty much copy & pasting another project into my project - maybe add a suggestion like `have a look at library ... it might do exactly what you need`?

**Comment by Urethramancer** [comment]
It's not copying anything directly - but sometimes great minds think alike ;)

A choice of preferred packages/libraries would be a great configuration option. Sometimes there are a lot of packages with the same names (particularly JSON and other data formats, different implementations for APIs and web middleware). And the biggest offender is SDKs with parts simply named "config" (looking at you, Amazon).
